subsys/event_manager/event_manager.c

```diff
@@ -30,9 +30,9 @@ static void event_processor_fn(struct k_work *work);
 #endif
 
 #ifdef CONFIG_SHELL
-extern uint32_t event_manager_displayed_events;
+extern struct event_manager_event_display_bm event_manager_event_display_bm;
```

Comment:
Maybe we could move it to separate file and expose simple API to keep the information in one place.?

Reply:
And for profiler in a separate file?